storage/innobase/xtrabackup/doc/source/innobackupex/streaming_backups_innobackupex.rst

```diff
@@ -1,59 +0,0 @@
-===================================
- Streaming and Compressing Backups
```

Comment:
this is partially covered by recipes_ibkx_stream.rst, not sure how to proceed with it

Reply:
Moved to xtrabackup_bin replacing references to innobackupex accordingly